consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/queries/QbftQueryServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.qbft.queries;
+
+import org.hyperledger.besu.consensus.common.BlockInterface;
+import org.hyperledger.besu.consensus.common.PoaQueryServiceImpl;
+import org.hyperledger.besu.consensus.common.bft.BftBlockHashing;
+import org.hyperledger.besu.consensus.common.bft.BftExtraData;
+import org.hyperledger.besu.crypto.NodeKey;
+import org.hyperledger.besu.ethereum.chain.Blockchain;
+import org.hyperledger.besu.ethereum.core.BlockHeader;
+import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.plugin.data.Address;
+import org.hyperledger.besu.plugin.services.query.BftQueryService;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.apache.tuweni.bytes.Bytes32;
+
+public class QbftQueryServiceImpl extends PoaQueryServiceImpl implements BftQueryService {
```

Comment:
Is this still needed? I thought this was replaced by the BftQueryServiceImpl

Reply:
true-  should be removed. ta.